src/imars3d/backend/util/functions.py

```diff
@@ -5,5 +5,11 @@
 import multiprocessing
 
 
-def clamp_max_workers(max_workers):
+def clamp_max_workers(max_workers) -> int:
```

Comment:
can we add type hinting for the input `max_workers` here, and add the standard docstring sections like the other function, i.e. `Parameters` and `Returns`.